support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx

```diff
@@ -20,6 +18,21 @@ import {
 } from './helpers';
 import { Collection } from 'helpers/productPrice/fulfilmentOptions';
 import { paperHasDeliveryEnabled } from 'helpers/subscriptions';
+import { Accordion, AccordionRow } from '@guardian/src-accordion';
+import { css } from '@emotion/core';
+import { neutral } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+
+
+const accordionContainer = css`
```

Comment:
Is this in the component file because it didn't seem worth moving to a separate styles file?

Reply:
TLDR; Yes
---------
If you want the long answer, it's this: if we were to separate this styling into separate files, we would have to have three separate files, one for each tab. There's a very strong principle with this sort of css that it must not be shared. In order to resist the temptation to share it, and because it was small, I have left it in the files. I know this is not your preference, and I'm happy to discuss if that would be helpful.